components/brave_extension/extension/brave_extension/_locales/en_GB/messages.json

```diff
@@ -63,6 +63,10 @@
     "message": "Scripts blocked",
     "description": "Message for the scripts blocked row label"
   },
+  "speedreaderActive": {
```

Comment:
btw only the source localization file en_US should be changed here, these are only written by pulls from transifex.

Reply:
moved to en_US, removed change in en_GB